src/components/ContentHighlights/CurrentContentHighlightHeader.jsx

```diff
@@ -2,22 +2,36 @@ import React, { useContext } from 'react';
 import {
   Button, ActionRow,
 } from '@edx/paragon';
+import { Add } from '@edx/paragon/icons';
 import ContentHighlightStepper from './HighlightStepper/ContentHighlightStepper';
 import { ContentHighlightsContext } from './ContentHighlightsContext';
 
 const CurrentContentHighlightHeader = () => {
   const {
     isModalOpen, setIsModalOpen,
   } = useContext(ContentHighlightsContext);
+
+  const handleNewClick = () => {
```

Comment:
@brobro10000 I don't think the `handleNewClick` function itself will go away, but the contents of the function (`setIsModalOpen`) would likely change to use `dispatch` from the reducer instead of this state setter function.